### PR TITLE
changes cult construct sprites

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

cult constructs now have new sprites, taken from /vg/ and modified, proteon and harvesters had big edits
theres now an animation and a sound for putting soulshards in shells
cult structures now use radials
new narsie sprites and spawning animation, more smooth rather than 8x resized small narsie sprite
also narsie can now turn mobs without clients into proteons in addition to wraiths juggernauts and artificers because that sounds cool 
i think deity link might have made the sprites but not sure

## Why It's Good For The Game

more otherwordly and magical sprites, they look really nice and radials are epic

## Changelog
:cl: Fikou, actioninja, /vg/ spriters (deitylink??)
tweak: cult structures use radials
imageadd: new sprites for cult constructs, stuff relating to them and new narsie sprites, taken from /vg/
/:cl:

### CODE REVIEW

**Comment by SkeletalElite** [comment]
What about purified constructs?

**Comment by Fikou** [comment]
> What about purified constructs?

what about them

**Comment by SkeletalElite** [comment]
> > What about purified constructs?
> 
> what about them

Did they get new sprites too?

**Comment by OverlyGeneric** [comment]
stray pixel in cult_s, make_wraith1, phase_shift2, and phase_shift

**Review by bobbah — CHANGES_REQUESTED** [review]
if this is what i think it is please document in your pr

File: code/modules/power/singularity/narsie.dm [hunk 1]
```diff
@@ -230,7 +231,7 @@
 	setDir(SOUTH)
 	move_self = 0
 	flick("narsie_spawn_anim",src)
-	sleep(11)
+	sleep(35)
```
Comment (Rohesie) [inline code/modules/power/singularity/narsie.dm:234]: `addtimer(CALLBACK(src, .proc/narsie_spawn_animation_end), 3.5 SECONDS)`

**Comment by Fikou** [comment]
-> #50620

**Pull request closed without merging**